website/source/docs/configuration/seal/awskms.html.md

```diff
@@ -116,6 +116,14 @@ Any new or updated data will be encrypted with the current key defined in the se
 or set to current under a key alias.
 
 
+## Recovery Key Rekeying
```

Comment:
This is more general than just for AWS.  I would suggest adding some of this documentation here https://www.vaultproject.io/docs/concepts/seal.html#seal-unseal.